diff -up ./tests/unit/test_client.py.orig ./tests/unit/test_client.py
--- ./tests/unit/test_client.py.orig	2021-07-15 14:54:46.760087937 -0500
+++ ./tests/unit/test_client.py	2021-07-15 14:56:01.748689317 -0500
@@ -16,7 +16,7 @@ import io
 import json
 import unittest
 
-import mock
+from unittest import mock
 
 
 def _make_credentials():
diff -up ./tests/unit/test__helpers.py.orig ./tests/unit/test__helpers.py
--- ./tests/unit/test__helpers.py.orig	2021-07-15 14:54:46.759087942 -0500
+++ ./tests/unit/test__helpers.py	2021-07-15 14:56:01.748689317 -0500
@@ -14,7 +14,7 @@
 
 import unittest
 
-import mock
+from unittest import mock
 
 
 class Test__LocalStack(unittest.TestCase):
diff -up ./tests/unit/test__http.py.orig ./tests/unit/test__http.py
--- ./tests/unit/test__http.py.orig	2021-07-15 14:55:02.892002183 -0500
+++ ./tests/unit/test__http.py	2021-07-15 14:56:01.749689311 -0500
@@ -17,7 +17,7 @@ import os
 import unittest
 import warnings
 
-import mock
+from unittest import mock
 import requests
 from six.moves import http_client
 
diff -up ./tests/unit/test_obsolete.py.orig ./tests/unit/test_obsolete.py
--- ./tests/unit/test_obsolete.py.orig	2021-07-15 14:54:46.760087937 -0500
+++ ./tests/unit/test_obsolete.py	2021-07-15 14:56:01.749689311 -0500
@@ -14,7 +14,7 @@
 
 import warnings
 
-import mock
+from unittest import mock
 
 from google.cloud import obsolete
 
