diff -up ./docs/conf.py.orig ./docs/conf.py
--- ./docs/conf.py.orig	2021-07-21 15:18:35.694535292 -0500
+++ ./docs/conf.py	2021-07-21 15:19:10.814414533 -0500
@@ -42,6 +42,7 @@ extensions = [
     "sphinx.ext.autodoc",
     "sphinx.ext.doctest",
     "sphinx.ext.intersphinx",
+    "sphinx_rtd_theme"
 ]
 
 # Add any paths that contain templates here, relative to this directory.
@@ -115,7 +116,7 @@ todo_include_todos = False
 
 # -- Options for HTML output ----------------------------------------------
 
-html_theme = "furo"
+html_theme = "sphinx_rtd_theme"
 html_theme_options = {}
 
 # Theme options are theme-specific and customize the look and feel of a theme
