From 2325721b997a540d7b4bd96541060b0c5234c777 Mon Sep 17 00:00:00 2001
From: Major Hayden <major@mhtx.net>
Date: Tue, 5 Oct 2021 06:50:17 -0500
Subject: [PATCH] Allow Python 3.10

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e751549..dcb63e1 100644
--- a/setup.py
+++ b/setup.py
@@ -133,7 +133,7 @@ setuptools.setup(
     namespace_packages=namespaces,
     install_requires=dependencies,
     extras_require=extras,
-    python_requires=">=3.6, <3.10",
+    python_requires=">=3.6",
     include_package_data=True,
     zip_safe=False,
 )
-- 
2.31.1

