diff -up ./tests/unit/gapic/functions_v1/test_cloud_functions_service.py.orig ./tests/unit/gapic/functions_v1/test_cloud_functions_service.py
--- ./tests/unit/gapic/functions_v1/test_cloud_functions_service.py.orig	2021-07-12 14:54:28.000000000 -0500
+++ ./tests/unit/gapic/functions_v1/test_cloud_functions_service.py	2021-07-21 14:39:13.660881830 -0500
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 import os
-import mock
+from unittest import mock
 import packaging.version
 
 import grpc
